sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -1588,10 +1587,9 @@ class SessionCatalog(
       TableFunctionRegistry.builtin.functionExists(name)
   }
 
-  protected[sql] def failFunctionLookup(
```

Comment:
Hmm, where do we set `cause`? As I see, we always call `failFunctionLookup` without `cause` (except for the test case).

Reply:
We used to pass the `cause` parameter before, when looking up builtin functions from Hive. The code path is gone now.